File: src/main/scala/resonantinduction/compatibility/nei/RISawmillRecipeHandler.java
package resonantinduction.compatibility.nei;

import resonant.lib.utility.LanguageUtility;
import resonant.content.factory.resources.RecipeType;

public class RISawmillRecipeHandler extends RITemplateRecipeHandler
{

	@Override
	public String getRecipeName()
	{
		return LanguageUtility.getLocal("resonantinduction.machine.sawmill");
	}

	@Override
	public RecipeType getMachine()
	{
		return RecipeType.SAWMILL;
	}
}

File: src/main/scala/resonantinduction/compatibility/nei/RISawmillRecipeHandler.scala
package resonantinduction.compatibility.nei

import resonant.lib.utility.LanguageUtility
import resonant.content.factory.resources.RecipeType

class RISawmillRecipeHandler extends RITemplateRecipeHandler
{
    def getRecipeName: String =
    {
        return LanguageUtility.getLocal("resonantinduction.machine.sawmill")
    }

    def getMachine: RecipeType =
    {
        return RecipeType.SAWMILL
    }
}